README.md

```diff
@@ -18,29 +18,57 @@ Insert screenshot of a demo page, including a code snippet.
 
 > Note: This guide assumes you have npm installed locally.
 
-To get started using one of the Material Web Components in your web application, simply:
+The easiest way to try out the Material Web Components is to use one of these online tools:
 
-1. Install each element you'd like to use:
+  * Runs in all [supported browsers](#browser-support): [Glitch](https://glitch.com/edit/#!/material-web-components)
```

Comment:
`No project found at material-web-components, perhaps a typo?`
Link not working.